CMakeLists.txt

```diff
@@ -279,28 +249,21 @@ if(NOT NONET)
   target_link_libraries(devilutionx PRIVATE sodium)
 endif()
 
-if(USE_SDL1)
-  target_link_libraries(devilutionx PRIVATE
-    ${SDL_TTF_LIBRARY}
-    ${SDL_MIXER_LIBRARY}
-    ${SDL_MIXER_LIBRARIES}
-    ${SDL_LIBRARY}
-    ${FREETYPE_LIBRARIES}
-    zlib)
-  if(WARPOS)
-    target_link_libraries(devilutionx PRIVATE -lmikmod)
+if(AMIGA)
+  if(USE_SDL1)
+    target_link_libraries(devilutionx PRIVATE
+      ${SDL_LIBRARY}
+      ${SDL_TTF_LIBRARY}
+      ${SDL_MIXER_LIBRARY}
+      ${FREETYPE_LIBRARIES})

```

Comment:
For some obscure reason, I can't just add this to the for each, or by itself?!?

Reply:
when linking static libraries, the link order is crucial. As an example, freetype needs to be linked in _after_ SDL_TTF